tars/tars.js

```diff
@@ -10,51 +10,65 @@
  * @return {Object}             Required package
  */
 function tarsRequire(packageName) {
-    var requirePackage;
+    var requiredPackage;
```

Comment:
I'd prefer to separate the code below into different functions to isolate scopes. Like, if `process.env.npmRoot`, then return result of one function, else return result of another. What do you think?